client/components/language-picker/README.md

```diff
@@ -0,0 +1,69 @@
+LanguagePicker
+==============
+
+React component used to display a Language Picker.
+
+---
+
+## Example Usage
+
+```js
+import LanguagePicker from 'components/language-picker';
+
+const languages = [
+	{
+		value: 1,
+		langSlug: 'en',
+		name: 'English',
+		wpLocale: 'en_US',
+		popular: 1
+	},
+	{
+		value: 11,
+		langSlug: 'cs',
+		name: 'Čeština',
+		wpLocale: 'cs_CZ'
+	}
+];
+
+class LanguagePickerDemo {
+	onSelectLanguage = ( event ) => {
+		this.setState( { language: event.target.value } );
+	}
+
+	render() {
+		return (
+			<LanguagePicker
+				languages={ languages }
+				valueKey="langSlug"
+				value={ this.state.language }
+				onChange={ this.onSelectLanguage }
+			/>
+		);
+	}
+}
```

Comment:
The example code in README now contains connecting to the `state/user-settings` Redux store :)